service/controller/appcatalog/v1/resource/index/current_test.go

```diff
@@ -0,0 +1,123 @@
+package index
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/micrologger/microloggertest"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func Test_Resource_GetCurrentState(t *testing.T) {
+	tests := []struct {
+		name              string
+		obj               interface{}
+		returnedConfigMap *corev1.ConfigMap
```

Comment:
```suggestion
		expectedConfigMap *corev1.ConfigMap
```

I'd prefer using expected as this is the expected state we're testing.